src/test/e2e/06_create_sbom_test.go

```diff
@@ -0,0 +1,62 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateSBOM(t *testing.T) {
+	cachePath := filepath.Join(os.TempDir(), ".cache-location")
+	sbomPath := filepath.Join(os.TempDir(), ".sbom-location")
+
+	e2e.cleanFiles(cachePath, sbomPath)
+
+	pkgName := fmt.Sprintf("zarf-package-dos-games-%s.tar.zst", e2e.arch)
+
+	stdOut, stdErr, err := e2e.execZarfCommand("package", "create", "examples/game", "--confirm", "--zarf-cache", cachePath, "--sbom-out", sbomPath)
+	require.NoError(t, err, stdOut, stdErr)
+	require.Contains(t, stdErr, "Creating SBOMs for 1 images and 0 components with files.")
+	// Test that the game package generates the SBOMs we expect (images only)
+	_, err = os.ReadFile(filepath.Join(sbomPath, "dos-games", "sbom-viewer-defenseunicorns_zarf-game_multi-tile-dark.html"))
+	require.NoError(t, err)
+	_, err = os.ReadFile(filepath.Join(sbomPath, "dos-games", "defenseunicorns_zarf-game_multi-tile-dark.json"))
+	require.NoError(t, err)
+
+	// Clean the SBOM path so it is force to be recreated
+	e2e.cleanFiles(sbomPath)
+
+	stdOut, stdErr, err = e2e.execZarfCommand("package", "inspect", pkgName, "--sbom-out", sbomPath)
+	require.NoError(t, err, stdOut, stdErr)
+	// Test that the game package generates the SBOMs we expect (images only)
+	_, err = os.ReadFile(filepath.Join(sbomPath, "dos-games", "sbom-viewer-defenseunicorns_zarf-game_multi-tile-dark.html"))
+	require.NoError(t, err)
+	_, err = os.ReadFile(filepath.Join(sbomPath, "dos-games", "defenseunicorns_zarf-game_multi-tile-dark.json"))
+	require.NoError(t, err)
+
+	version, stdErr, err := e2e.execZarfCommand("version")
```

Comment:
It would be nice to add a comment to why we need version here (see it below, but I totally at first was like, what is he doing??)

Reply:
done